ports/ttauri/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_fail_port_install(ON_ARCH "x86" "arm" "arm64")
+vcpkg_fail_port_install(ON_TARGET "linux" "uwp" "osx")
+vcpkg_fail_port_install(ON_LIBRARY_LINKAGE "dynamic")
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO ttauri-project/ttauri
+    REF v0.3.0
+    SHA512 8e01ea28516063902483da3fae1ecf8524d47803b3809c289ce6bba39fd47e6ba20d8882f2cfce9a0f7101b917f51659592dfe38f0353f91977e6db7f94e0400
+    HEAD_REF main
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS -DTT_BUILD_TESTS=OFF -DTT_BUILD_EXAMPLED=OFF
```

```suggestion
    OPTIONS
        -DTT_BUILD_TESTS=OFF
        -DTT_BUILD_EXAMPLES=OFF
        -DCMAKE_DISABLE_FIND_PACKAGE_Doxygen=ON
```
`EXAMPLED` was misspelled. Additionally, we don't want doc generation to be sensitive to system state.

---

You have a good eye :-)